pkg/services/sqlstore/org_users.go

```diff
@@ -207,8 +208,8 @@ func (ss *SQLStore) SearchOrgUsers(query *models.SearchOrgUsersQuery) error {
 	return nil
 }
 
-func RemoveOrgUser(cmd *models.RemoveOrgUserCommand) error {
-	return inTransaction(func(sess *DBSession) error {
+func (ss *SQLStore) RemoveOrgUser(ctx context.Context, cmd *models.RemoveOrgUserCommand) error {
+	return inTransactionCtx(ctx, func(sess *DBSession) error {
```

Comment:
Better to use this when you have `*SQLStore` instance available since it will not use the global `x` xorm engine.

```suggestion
	return ss.WithTransactionalDbSession(ctx, func(sess *DBSession) error {
```